consumers/src/main/java/uk/gov/moj/sdt/cmc/consumers/client/mock/MockJudgementServiceRequestService.java

```diff
@@ -0,0 +1,22 @@
+package uk.gov.moj.sdt.cmc.consumers.client.mock;
+
+import java.util.Calendar;
+
+import org.springframework.stereotype.Service;
+import uk.gov.moj.sdt.cmc.consumers.api.IJudgementService;
+import uk.gov.moj.sdt.cmc.consumers.request.judgement.JudgementRequest;
+import uk.gov.moj.sdt.cmc.consumers.response.judgement.JudgementResponse;
+
+@Service("MockJudgementRequestService")
+public class MockJudgementServiceRequestService implements IJudgementService {
```

Comment:
Should this be named `MockJudgementRequestService` rather than `MockJudgementServiceRequestService`?

Reply:
done